presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -871,6 +871,33 @@ public void testDistinctHaving()
                 "GROUP BY orderdate " +
                 "HAVING COUNT(DISTINCT clerk) > 1");
     }
+    @Test
+    public void testSelectFilter()
+            throws Exception
+    {
+        MaterializedResult actual = computeActual(
+                "SELECT SUM(X) FILTER(WHERE Y>4) FROM  " +
+                " (VALUES (1,3),(2,4),(2,4),(4,5)) t(X,Y)");
+        MaterializedResult expected = resultBuilder(getSession(), BIGINT).row(4L).build();
+        assertEquals(actual, expected);
+        actual = computeActual(
```

Comment:
This formatting isn't correct. The IntelliJ formatter will change it when using our code style: https://github.com/airlift/codestyle

It needs to be written like this:
```java
computeActual("" +
        "SELECT ... " +
        "FROM ...");